recipe/bld.bat

```diff
@@ -13,33 +13,39 @@ cmake -G "Ninja" ^
       -DARROW_BUILD_STATIC=OFF ^
       -DARROW_BUILD_TESTS=OFF ^
       -DARROW_BUILD_UTILITIES=OFF ^
+      -DARROW_COMPUTE=ON ^
       -DARROW_CSV=ON ^
       -DARROW_CUDA=OFF ^
       -DARROW_DATASET=ON ^
       -DARROW_DEPENDENCY_SOURCE=SYSTEM ^
+      -DARROW_FILESYSTEM=ON ^
       -DARROW_FLIGHT=ON ^
       -DARROW_FLIGHT_REQUIRE_TLSCREDENTIALSOPTIONS=ON ^
       -DARROW_GANDIVA=OFF ^
       -DARROW_HDFS=ON ^
       -DARROW_JSON=ON ^
       -DARROW_MIMALLOC=OFF ^
       -DARROW_OPENSSL_USE_SHARED=ON ^
-      -DARROW_ORC=ON ^
+      -DARROW_ORC=OFF ^
       -DARROW_PACKAGE_PREFIX="%LIBRARY_PREFIX%" ^
       -DARROW_PARQUET=ON ^
       -DARROW_PLASMA=ON ^
       -DARROW_PROTOBUF_USE_SHARED=ON ^
-      -DARROW_PYTHON=ON ^
+      -DARROW_PYTHON=OFF ^
```

Comment:
What about `DARROW_PYTHON` flag in `build.sh`? It has ON there.

Reply:
For some reason the `DARROW_PYTHON` flag seems to create problems in windows. Since it is deprecated we probably should disable it in all architectures.

Reply:
It's deprecated but it can still work

Reply:
@chenghlee @varlackc Here is how conda-forge synced build scripts with upstream (option additions/removals) for v10.0.0 https://github.com/conda-forge/arrow-cpp-feedstock/commit/ee1177dc0b2415fe26a6e702794b0e1079343100.

Reply:
the  `DArrow_Python` option has been added and turned on.